File: OctobridgeEvaluation/Angular.Client/src/app/models/product.ts
//
// Product Class
//
export class Product {
  productId: number;
  name: string;
  productNumber: string;
  productCategoryId: number;
  cost: number;
  recordCreated: Date | null;
  recordModified: Date | null;

  constructor(theProductId = (-1), theName = '', theProductNumber = '', theProductCategoryId = (-1), theCost = 0,
    theRecordCreated: Date | null = null, theRecordModified: Date | null = null) {
    this.productId = theProductId;
    this.name = theName;
    this.productNumber = theProductNumber;
    this.productCategoryId = theProductCategoryId;
    this.cost = theCost;
    this.recordCreated = theRecordCreated;
    this.recordModified = theRecordModified;
  }

}

File: OctobridgeLegacy/DataMaintenanceEnhanced/Product.cs
namespace DataMaintenanceEnhanced;

public class Product
{
    public int ProductId { get; set; }
    public string Name { get; set; }
    public string ProductNumber { get; set; }
    public int ProductCategoryId { get; set; }
    public decimal Cost { get; set; }
    public DateTime RecordCreated { get; set; }
    public DateTime RecordModified { get; set; }

    public Product()
    {
    }
}

